x-pack/metricbeat/module/sql/query/test_sql_mssql.py

```diff
@@ -0,0 +1,62 @@
+import os
+import sys
+import unittest
+import time
+from xpack_metricbeat import XPackTest, metricbeat
+
+
+class Test(XPackTest):
+    COMPOSE_SERVICES = ['mssql']
+
+    @unittest.skipUnless(metricbeat.INTEGRATION_TESTS, "integration test")
+    def test_query_without_fetch_from_all_databases(self):
+        """
+        SQL MSSQL custom query with fetch_from_all_databases=False
+        """
+        fetch_from_all_databases = False
+        self.run_query_test(fetch_from_all_databases)
+
+    @unittest.skipUnless(metricbeat.INTEGRATION_TESTS, "integration test")
+    def test_query_with_fetch_from_all_databases(self):
+        """
+        SQL MSSQL custom query with fetch_from_all_databases=True
+        """
+        fetch_from_all_databases = True
+        self.run_query_test(fetch_from_all_databases)
```

Comment:
```suggestion
    @unittest.skipUnless(metricbeat.INTEGRATION_TESTS, "integration test")
    def test_query_without_fetch_from_all_databases(self):
        """
        SQL MSSQL custom query with fetch_from_all_databases=false
        
        1 document will be received.
        """
        self.run_query_test(fetch_from_all_databases=False, expected_output_count=1)

    @unittest.skipUnless(metricbeat.INTEGRATION_TESTS, "integration test")
    def test_query_with_fetch_from_all_databases(self):
        """
        SQL MSSQL custom query with fetch_from_all_databases=true
        
        4 documents will be received because 4 documents are present by default in MSSQL.
        """
        self.run_query_test(fetch_from_all_databases=True, expected_output_count=4)

```
Looks good! Minor nits from my side:

1) Since you are still querying the DB with lower case `false`, I would keep it that way in the docstring even if you are passing `False`, your choice here 😄 

2) You could use `self.run_query_test(fetch_from_all_databases=False/True)` without initializing the variable beforehand.

3) Sorry to give you more work here, but since you are also documenting what `expected_output_count` is, it could be a good idea to just pass it to `run_test_query` directly (and make changes accordingly) to enhance the docstring. WDYT @shmsr ?

Reply:
Thanks a lot, Lucian. The changes you suggested definitely make it nicer. 

I just made a couple of more docstring changes.